### PR TITLE
Added recipe for msg-extractor

### PR DESCRIPTION
**Pull request by pmlandwehr**
[`msg-extractor`](https://github.com/mattgwwalker/msg-extractor) is a funky package since it isn't a module and isn't on pypi. The primary issue with this is that we can't easily patch it to work with an `entry_point`, so I've left that out of the build. It likely won't be possible to invoke the package from the command line, but it will still be usable in imports.

### CODE REVIEW

**Comment by pmlandwehr** [comment]
Assuming this failed because all of the windows modules weren't yet up. Closing/reopening

**Comment by pmlandwehr** [comment]
Well, I'm really not sure what to make of this - the module can't find `olefile` on `win and x86 and (py27 or py35)` which is a very strange combination. Any ideas, @jakirham or others?

**Comment by jakirkham** [comment]
Seems those packages were broken. See issue ( https://github.com/conda-forge/olefile-feedstock/issues/1 ).

**Comment by pmlandwehr** [comment]
Closing and reopening, now that `olefile` is patched

**Comment by pmlandwehr** [comment]
@jakirkham Hooray! Can we get a merge?

File: recipes/msg-extractor/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{%set name = "msg-extractor" %}
+{%set version = "0.2" %}
+{%set build_num = "0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/mattgwwalker/{{ name }}.git
```
Comment (ocefpaf) [inline recipes/msg-extractor/meta.yaml]: Can we do this here instead?

```yaml
  fn: {{ name }}-{{ version }}.tar.gz
  url: https://github.com/mattgwwalker/msg-extractor/archive/v{{ version }}.tar.gz
  sha256: 9189ab65c801c0269534e3ffb4f5923a02c8766d2b21e9412c6fa885e77e7fdb
```

**Comment by jakirkham** [comment]
LGTM. @ocefpaf?

File: recipes/msg-extractor/meta.yaml (hunk 1 above)
Comment (pmlandwehr) [inline recipes/msg-extractor/meta.yaml]: (Alas, didn't work)

**Comment by pmlandwehr** [comment]
(ping @ocefpaf for merge)

**Pull request merged**